bookmarks.html

```diff
@@ -4,8 +4,79 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Bookmarks</title>
     <link rel="stylesheet" href="/styles/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+      integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+      crossorigin="anonymous"
+    />
   </head>
-  <body></body>
+  <body>
+    <h1>Bookmarks</h1>
+    <main class="questions">
+      <div class="card">
+        <h2 class="question-header mx-1rem">Question:</h2>
+        <p class="question mx-1rem">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Facilis
+          quaerat asperiores, eius alias doloribus ut doloremque laudantium
+          soluta ipsum enim!
+        </p>
+        <button class="buttonTurnCard">Show Answer</button>
+        <ul class="tagList">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JS</li>
+          <li>Clean Code</li>
+        </ul>
+      </div>
+      <div class="card">
+        <h2 class="question-header mx-1rem">Question:</h2>
+        <p class="question mx-1rem">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Facilis
+          quaerat asperiores, eius alias doloribus ut doloremque laudantium
+          soluta ipsum enim!
+        </p>
+        <button class="buttonTurnCard">Show Answer</button>
+        <ul class="tagList">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JS</li>
+          <li>Clean Code</li>
+        </ul>
+      </div>
+      <div class="card">
+        <h2 class="question-header mx-1rem">Question:</h2>
+        <p class="question mx-1rem">
+          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Facilis
+          quaerat asperiores, eius alias doloribus ut doloremque laudantium
+          soluta ipsum enim!
+        </p>
+        <button class="buttonTurnCard">Show Answer</button>
+        <ul class="tagList">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JS</li>
+          <li>Clean Code</li>
+        </ul>
+      </div>
+    </main>
+    <nav>
+      <ul class="navigation" role="list">
+        <li class="nav-item">
+          <a href="index.html"><i class="fas fa-home"></i></a>
```

Comment:
The anchors need some text to be accessible. You can hide it with a "screenreader only css class" (google it).